configuration/locationtags/locationtags.csv

```diff
@@ -16,6 +16,7 @@ d9865139-dfb4-11e4-bccc-56847afe9799,,Vitals Location,Signifies a Location where
 5e824f3a-dfb8-11e4-bccc-56847afe9799,,Inpatients App Location,Signifies a Location where the inpatients app should be available
 24ffeea6-dfbb-11e4-bccc-56847afe9799,,Check-In Location,Signifies a Location where the check-in app and form should be available
 8cff42e5-f03b-461d-b693-65bdf10c4bba,,Check-In Maternal Location,Location where the maternal check-in app and form is available
+a61fa90a-d71d-4b44-b148-affac16fdec3,,MCOE Triage Location,Location where the MCOE triage form is available
```

Comment:
I thought the MCOE Triage Location tag would be added to the sl config, not here.  It's already a location in the sl configuration.

Reply:
That's what I thought too @lnball . But there is no locationtag config in the pishsl, and whoever added the Check-In Maternal Location tag, has also added that tag in this same file. Please see the one line above. So, I just wanted to follow the same pattern. Thanks!

Reply:
"MCOE" is such a specific term to that implementation, I do agree that it would make sense to create a new location tags file for SL, and add it there instead... *but* an alternatively could be to just rename this "MCH Triage Location" or "Maternal Triage Location" (to be consistent with "Check-In Maternal".

Reply:
Thanks @lnball and @mogoodrich ! I have renamed to MCH Triage Location